contracts/gateway/OSTPrime.sol

```diff
@@ -170,7 +170,7 @@ contract OSTPrime is UtilityToken, OSTPrimeConfig, Mutex {
         );
 
         /*
-         * The OST Prime base token balance of contract should always be
+         * The OST Prime coin balance of contract should always be
```

Comment:
I don't understand the whole comment 🤔 Maybe you can rewrite it in a clearer way?